app/Http/Controllers/DemoController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Room;
+
+class DemoController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return inertia('Demo/Tables/RoomTableIndex', [
+            'rooms' => Room::all(),
+        ]);
+    }
```

Comment:
these lines aren't covered by any tests. Please create a test before merging.

Reply:
Added tests for this, resolving comment now